### PR TITLE
Adding support for editing schedules

### PR DESCRIPTION
**Pull request by Bpoe**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by Bpoe** [comment]
No automated tests have been included in this PR but we have been using this code in production for some time and we have automated coverage for it as part of our GQL and DVT tests on our side.

File: src/ResourceManagement/Automation/AutomationManagement/Generated/Models/ScheduleCreateOrUpdateParameters.cs [hunk 1]
```diff
@@ -30,6 +30,18 @@ namespace Microsoft.Azure.Management.Automation.Models
     /// </summary>
     public partial class ScheduleCreateOrUpdateParameters
     {
+        private bool _convertTimesFromTimeZone;
+        
+        /// <summary>
+        /// Optional. True if times provided are specified in the provided time
+        /// zone. False if times are in UTC.
+        /// </summary>
+        public bool ConvertTimesFromTimeZone
```
Comment (shahabhijeet) [inline src/ResourceManagement/Automation/AutomationManagement/Generated/Models/ScheduleCreateOrUpdateParameters.cs]: @Bpoe can you add link to the hydra spec PR to this PR

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Once the hydra spec PR is linked, this PR is good to go.

File: src/ResourceManagement/Automation/AutomationManagement/Generated/Models/ScheduleCreateOrUpdateParameters.cs (hunk 1 above)
Comment (Bpoe) [inline src/ResourceManagement/Automation/AutomationManagement/Generated/Models/ScheduleCreateOrUpdateParameters.cs]: https://github.com/Azure/hydra-specs-pr/pull/1396

**Comment by Bpoe** [comment]
@shahabhijeet The link to the hydra spec PR was added. Is there anything else that is blocking this from being merged?

**Pull request merged**